### PR TITLE
lune 0.7.5 (new formula)

### PR DESCRIPTION
**Pull request by jackprogramsjp**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/lune.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  desc "Standalone Luau script runtime"
+  homepage "https://lune-org.github.io/docs"
+  url "https://github.com/filiptibell/lune/archive/refs/tags/v0.7.5.tar.gz"
+  sha256 "e8191df5d6844026772cc7afab1083235a265c506474c4c4dee0a7724b04f775"
+  license "MPL-2.0"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/lune.rb:11]: yeah, I think this is removable.

[the remark this answers is not in the document]
Comment (jackprogramsjp) [reply Formula/lune.rb:11]: So, I can't add a livecheck to auto update it?

[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/lune.rb:11]: Well, you can use the default livecheck.

File: Formula/lune.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"test.lua").write("print(2 + 2)")
+    output = shell_output("#{bin}/lune test.lua")
+    assert_equal "4", output.chomp
```
[the remark this answers is not in the document]
Comment (jackprogramsjp) [reply Formula/lune.rb]: Ok, it's good.

File: Formula/lune.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (jackprogramsjp) [reply Formula/lune.rb:11]: Oh ok, yeah, then that would work.

**Comment by jackprogramsjp** [comment]
I don't mean to be annoying, but I have already finished everything for this, so am I supposed to close this PR now so then you guys know it's ready for merge?

**Comment by jackprogramsjp** [comment]
Put in the wrong flag, now it should be fixed the CI should not fail.

**Comment by chenrui333** [comment]
@jackprogramsjp can you squash all the commits into one with commit msg as `lune 0.7.5 (new formula)`? Thanks!

**Review by chenrui333 — APPROVED** [review]

**Comment by SMillerDev** [comment]
> I hate to be annoying, but I have already finished everything for this, so am I supposed to close this PR now so then you guys know it's ready for merge?

You just wait until a volunteer has time to review it.

**Comment by jackprogramsjp** [comment]
@chenrui333 I have squashed it all into one commit.

**Comment by jackprogramsjp** [comment]
I totally renamed the commit message wrong, I will do that.

**Comment by jackprogramsjp** [comment]
@chenrui333 @SMillerDev Changed the commit messages, now it should be ready!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@jackprogramsjp, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**